Kickstarter-iOS/Features/DesignSystem/Colors/DesignSystemBackgroundColors.swift

```diff
@@ -0,0 +1,95 @@
+import UIKit
+
+public enum DesignSystemBackgroundColors {
+  case backgroundSurfacePrimary
+  case backgroundSurfaceInverse
+  case backgroundDisabled
+  case backgroundInverse
+  case backgroundInverseHover
+  case backgroundInversePressed
+  case backgroundSelected
+  case backgroundAction
+  case backgroundActionHover
+  case backgroundActionDisabled
+  case backgroundActionPressed
+  case backgroundAccentGreenBold
+  case backgroundAccentGreenSubtle
+  case backgroundAccentBlueBold
+  case backgroundAccentBlueSubtle
+  case backgroundAccentPurpleSubtle
+  case backgroundDangerBold
+  case backgroundDangerSubtle
+  case backgroundDangerBoldPressed
+  case backgroundDangerBoldHovered
+  case backgroundDangerSubtleHovered
+  case backgroundAccentGrayBold
+  case backgroundAccentGraySubtle
+  case backgroundWarningBold
+  case backgroundWarningSubtle
+
+  var rawValue: String {
+    switch self {
+    case .backgroundSurfacePrimary:
+      return DesignSystemCoreColors.white.rawValue
+    case .backgroundSurfaceInverse:
+      return DesignSystemCoreColors.grey10.rawValue
+    case .backgroundDisabled:
+      return DesignSystemCoreColors.grey4.rawValue
+    case .backgroundInverse:
```

Comment:
What does backgroundInverse mean? This is the same as the backgroundSurface color.

Reply:
Naming and colors are all taken directly from the web right now. Not all of them apply to our current design system. The intention here is to give us a good starting point (common ground) for working with Designer Matthew when he comes on board. Once dark mode designs are defined, it _should be_ easier to update these existing assets' colors and names to be more mobile-friendly and relevant than doing so from scratch. At least, that's how Android and management decided to approach this. I thought that was fine. 

I can see an argument for only adding color definitions like this once they have been defined by design.